recipes/r-akima/LICENSE

```diff
@@ -0,0 +1,90 @@
+terms of this Agreement.  Noncommercial Use
+
+The ACM grants to you (hereafter, User) a royalty-free, nonexclusive right to
+execute, copy, modify and distribute both the binary and source code solely for
+academic, research and other similar noncommercial uses, subject to the
+following conditions:
+
+    User acknowledges that the Software is still in the development stage and
+    that it is being supplied "as is," without any support services from
+    ACM. Neither ACM nor the author makes any representations or warranties,
+    express or implied, including, without limitation, any representations or
+    warranties of the merchantability or fitness for any particular purpose, or
+    that the application of the software, will not infringe on any patents or
+    other proprietary rights of others.
+
+    ACM shall not be held liable for direct, indirect, incidental or
+    consequential damages arising from any claim by User or any third party with
+    respect to uses allowed under this Agreement, or from any use of the
+    Software.
+
+    User agrees to fully indemnify and hold harmless ACM and/or the author(s) of
+    the original work from and against any and all claims, demands, suits,
+    losses, damages, costs and expenses arising out of the User's use of the
+    Software, including, without limitation, arising out of the User's
+    modification of the Software.
+
+    User may modify the Software and distribute that modified work to third
+    parties provided that: (a) if posted separately, it clearly acknowledges
+    that it contains material copyrighted by ACM (b) no charge is associated
+    with such copies, (c) User agrees to notify ACM and the Author(s) of the
+    distribution, and (d) User clearly notifies secondary users that such
+    modified work is not the original Software.
+
+    User agrees that ACM, the authors of the original work and others may enjoy
+    a royalty-free, non-exclusive license to use, copy, modify and redistribute
+    these modifications to the Software made by the User and distributed to
+    third parties as a derivative work under this agreement.
+
+    This agreement will terminate immediately upon User's breach of, or
+    non-compliance with, any of its terms. User may be held liable for any
+    copyright infringement or the infringement of any other proprietary rights
+    in the Software that is caused or facilitated by the User's failure to abide
+    by the terms of this agreement.
+
+    This agreement will be construed and enforced in accordance with the law of
+    the state of New York applicable to contracts performed entirely within the
+    State. The parties irrevocably consent to the exclusive jurisdiction of the
+    state or federal courts located in the City of New York for all disputes
+    concerning this agreement.
+
+Commercial Use
+
+Any User wishing to make a commercial use of the Software must contact ACM at
```

Comment:
I'm not sure what the police for conda-forge is, but I have some problems with this. Especially if this is a dependency of other tools and you actually don't know about this restriction.

Reply:
@jakirkham any police in place?

Reply:
Conda-forge is not a commercial (or even a legal) entity, and so this restriction does not apply to us.  If someone takes the conda-forge package and redistributes it for commercial gain, they would be the ones in breech of the license, not conda-forge.

Also if someone redistributes the conda-forge package commercially, they were provided the license in the package telling them not to.  

I don't see any reason for conda-forge to not build this package nor any harm to conda-forge in doing so.

Reply:
I think you are probably right. Though IANAL.

Would add that when I see non-standard license text like this, I kind of wish conda-forge did have some sort of legal counsel so they could review and advise.

Reply:
@jakirkham - yeah that would be one benefit of having some sort of numfocus affiliation

Reply:
I agree with everything that has been said so far:

1. This license is more restrictive than most users would expect. From my reading, even if the commercial product makes the source code openly available (e.g. as required by a copyleft style license), they still have to obtain permission first from ACM.
1. conda-forge is not violating the license because we are not including it in a commercial product, and we are including the license with the distributed binary.
1. The point of including the license information with the conda recipe and conda binary is so that end users can decide if the software license is appropriate for their use case.
1. Most end users are unlikely to check the licenses of every dependency (but also most users are not creating a commerical product, and those that are should seek legal counsel)
1. Unfortunately none of us our lawyers...

I investigated the packages that have akima as a dependency. There are 4 CRAN packages (DATforDCEMRI, FGN, RchivalTag, spikeSlabGAM), and although they don't state anything about the ACM license, they are all CC BY-NC-SA (i.e. non-commercial, so that is somewhat compliant).

There are 3 Bioconductor packages (LedPred, OCplus, splatter). These are much more troublesome. LedPred is MIT, OCplus is LGPL, and splatter is GPL3. From my understanding, all 3 packages are violating the terms of the ACM license by stating their packages are available under these open source licenses.

Reply:
Does anyone else have an opinion on the license for this package? What is the process for making a final decision on a recipe with a non-standard license?

Reply:
I think that this is probably OK to merge.  I'll be happy to do it if no one objects in the next 8 hrs